style.css

```diff
@@ -0,0 +1,97 @@
+body {
+  background-color: #e5e5e5;
+  margin: 0;
+}
+
+.starting-page {
+  background-color: #fff;
+  border-radius: 0 0 0 5%;
+}
+
+#navigator_bar {
+  display: flex;
+}
+
+.navigator_menu {
+  width: 45px;
+  height: 30px;
+  display: flex;
+  align-items: center;
+  margin-top: 20px;
+}
+
+#navigator_bar nav:hover {
+  cursor: pointer;
+}
+
+.menu-link {
+  display: none;
+}
+
+#section-presentation {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 400px;
+}
+
+.short-presentation {
+  font-weight: bold;
+  font-size: 40px;
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design

  - You have no background image.
  - You have horizontal overflow
  - Your font style, size, and family do not match Figma the design
  
  ### YOUR DESIGN
  
  [Screencast from 02-16-2023 11:58:59 AM.webm](https://user-images.githubusercontent.com/92303891/219359197-e274a17c-a4b6-4d5b-8e5c-8e32b322029b.webm)
   
  ### FIGMA DESIGN
  
  | ![image](https://user-images.githubusercontent.com/92303891/219358880-400241d3-1858-453a-8138-c295926b387d.png)|

Reply:
Resolved the background issue, font issue, horizontal overflow